packages/clerk-js/src/ui/components/UserProfile/ActiveDevicesSection.tsx

```diff
@@ -154,11 +165,49 @@ const DeviceInfo = (props: { session: SessionWithActivitiesResource }) => {
         <Text
           variant='smallRegular'
           colorScheme='neutral'
-          sx={{ textTransform: 'capitalize' }}
         >
-          {formatRelative(props.session.lastActiveAt || new Date(), new Date())}
+          {getRelativeToNowDate(props.session.lastActiveAt, t)}
         </Text>
       </Col>
     </Flex>
   );
 };
+
+type RelativeDateKeys =
```

Comment:
We can avoid duplicating these by slightly changing getRelativeToNowDate, see comment below